src/components/Reports/SubscriberHistory/SubscriberHistory.js

```diff
@@ -28,6 +28,10 @@ const getDeliveryStatusCssClassName = (deliveryStatus) => {
   return deliveryCssClass;
 };
 
+const searchLinkByRel = (items, rel) => {
+  return items.find((x) => x.rel.includes(rel));
```

Comment:
Great, in the future I think that _searchLinkByRel_ could be improved to be more robust, for example, splitting the string using spaces, but by the moment it is fine and it is easy to modify.